nullaway/src/main/java/com/uber/nullaway/handlers/ApacheThriftIsSetHandler.java

```diff
@@ -54,17 +54,26 @@ public class ApacheThriftIsSetHandler extends BaseNoOpHandler {
 
   private static final Supplier<Type> TBASE_TYPE_SUPPLIER = Suppliers.typeFromString(TBASE_NAME);
 
-  @Nullable private Optional<Type> tbaseType;
+  private Optional<Type> tbaseType;
 
   @Override
   public void onMatchTopLevelClass(
       NullAway analysis, ClassTree tree, VisitorState state, Symbol.ClassSymbol classSymbol) {
     if (tbaseType == null) {
-      tbaseType =
-          Optional.ofNullable(TBASE_TYPE_SUPPLIER.get(state)).map(state.getTypes()::erasure);
+      init(state);
     }
   }
 
+  /**
+   * This method is annotated {@code @Initializer} since it will be invoked when the first class is
+   * processed by {@link #onMatchTopLevelClass(NullAway, ClassTree, VisitorState,
+   * Symbol.ClassSymbol)}
+   */
+  @Initializer
```

Comment:
Just out of curiosity, was there any issue marking `onMatchTopLevelClass(...)` itself as an initializer? Or is there a reason why that would be confusing? (I guess for all handlers, in general, we would expect that to be the first callback executed always?)

Reply:
You're right, just annotating `onMatchTopLevelClass(...)` is better, and I switched to that.  It's slightly strange since that method gets invoked multiple times (once for each class), but it does perform guaranteed initialization the first time it runs, so I think it's fine to annotate it as `@Initializer`.